bindings/binding_typescript_wasm/.cargo/config.toml

```diff
@@ -0,0 +1,3 @@
+[target.wasm32-unknown-unknown]
+# support more stack size due to issue: https://github.com/swc-project/swc/issues/10207
```

Comment:
We can using another way to resolve this case: iterative parsing of parenthesis.
This PR can be a alternative soluation.